sdk/Metrics/AbstractExporter.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Sdk\Metrics;
+
+use Exception;
+use OpenTelemetry\Metrics as API;
+use OpenTelemetry\Sdk\Metrics\Exceptions\CantBeExported;
+use OpenTelemetry\Sdk\Metrics\Exceptions\RetryableExportException;
+
+abstract class AbstractExporter implements API\Exporter
+{
+    /**
+     * {@inheritDoc}
+     */
+    public function export(iterable $metrics): int
+    {
+        // todo: we need to implement Logger?
```

Comment:
The logger project will come after the Tracing and Metrics API/SDK pairs are completed.  They are still working through the implementations.  I think this isn't a primary concern right now.